src/App.jsx

```diff
@@ -11,7 +12,10 @@ function App() {
     <Header />
     <Routes>
       <Route path='/' element={<ArticlesList />} />
+      <Route path='/articles' element={<ArticlesList />} />
+      <Route path='/articles/:topic' element={<ArticlesList />} />
```

Comment:
We're using "/articles?topic=(topic)" so this route /articles/:topic is not being used. Can remove that